mmdeploy/codebase/mmdet/models/dense_heads/rtmdet_head.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional
+
+import torch
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+from torch import Tensor
+
+from mmdeploy.codebase.mmdet import get_post_processing_params
+from mmdeploy.codebase.mmdet.models.layers import multiclass_nms
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
```

Comment:
Since RTMDetHead share the same `predict_by_feat` with base_dense_head, we can export the model without this rewriter. Is there any optimization in this rewriter?

Reply:
We use the same logic in YOLOX's predict_by_feat for a fair comparison with YOLO series.